other/dns-policy-and-dns-config/dns-policy-and-dns-config.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: dns-policy
+  annotations:
+    policies.kyverno.io/title: Change DNS Config and Policy
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.7.3
```

Comment:
This annotation holds the version of Kyverno you actually tested and verified operation. Is that actually 1.7.3? Can you check and certify on Kyverno 1.8.1?

Reply:
Actually, it works with both 1.7.3 and 1.8.1; I checked it.